api/oss/src/core/sessions/dtos.py

```
@@ -5,9 +5,10 @@
 
 from oss.src.core.sessions.records.dtos import SessionMessagePreview
 from oss.src.core.sessions.streams.dtos import SessionStream, SessionStreamQueryFlags
-from oss.src.core.shared.dtos import Reference
+from oss.src.core.sessions.types import ReferenceKey as ReferenceKey
```

Flake8 (7.3.0)

[error] 'oss.src.core.sessions.types.ReferenceKey' imported but unused

(F401)